### DISCUSSION TITLE
Removing Download button in item page

### DISCUSSION THREAD

**Discussion by vgar16** [CollectionBuilder/collectionbuilder.github.io#132] [Q&A]
Hello, 

Can the download button be removed from the item page? If so, how do I format the page to keep only the "View on Map" button? 

There is a previous Q&A in response to this question, but the answer given does not apply anymore because the code appears to have changed since. 

Thank you for your time and response.

**Comment by evanwill** [comment]
@vgar16 which template are you using? There is a difference between how CB-GH/-SHEETS and CB-CSV handle the item pages. 

It looks like you are using CB-Sheets so I will start with details for that:

CB-GH and CB-Sheets, all the "Item" page displays are really one page "/item.html". The display of the items is controlled by the javascript in the file "_includes/js/item-js.html"--so unfortunately, it is not super easy to customize, but is possible! 
Open your "_includes/js/item-js.html", and look for line 136 that looks like:

`itemLink += item.format.includes('compound_object') || item.format.includes('multiple') ? compoundDownload : itemDownload;`

Comment out that line by adding two slashes in front, like:

`// itemLink += item.format.includes('compound_object') || item.format.includes('multiple') ? compoundDownload : itemDownload;`

That will remove the Item download button, but keep the other buttons.

**Reply by vgar16** [reply]
Hello @evanwill ,

Thank you so much that worked!
